app-config.yaml

```diff
@@ -305,6 +305,8 @@ auth:
           $env: AUTH_OAUTH2_AUTH_URL
         tokenUrl:
           $env: AUTH_OAUTH2_TOKEN_URL
+        scope:
+          $env: AUTH_OAUTH2_SCOPE
```

Comment:
No need to put this in an env var, could just add a commented out example value here instead like in the changeset